contracts/contracts/coordination/TACoChildApplication.sol

```diff
@@ -61,18 +61,21 @@ contract TACoChildApplication is ITACoRootToChild, ITACoChildApplication, Initia
     /**
      * @notice Initialize function for using with OpenZeppelin proxy
      */
-    function initialize(address _coordinator, address _adjudicator) external initializer {
-        require(coordinator == address(0) || adjudicator == address(0), "Contracts already set");
```

Comment:
@vzotova why the OR here? Shouldn't this have an AND?

Reply:
because in mainnet coordinator already initialized, so allows to use this function if previous version was deployed before